payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CvcConfig.kt

```diff
@@ -22,7 +22,10 @@ internal class CvcConfig : CardDetailsTextFieldConfig {
         return if (number.isEmpty()) {
             TextFieldStateConstants.Error.Blank
         } else if (brand == CardBrand.Unknown) {
-            TextFieldStateConstants.Error.Invalid(R.string.invalid_card_number)
+            when (number.length) {
```

Comment:
I think if the card brand is unknown the field will always be limitless, no?

Reply:
That means the field wouldn't advance to the next field after 4 digits, which would be inconsistent with master.

Reply:
ok, we can leave it, thanks for checking with master.